Setup/Patch/Data/AddFavouriteFilm.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Wtc\Film\Setup\Patch\Data;
+
+use Magento\Customer\Model\Customer;
+use Magento\Eav\Setup\EavSetup;
+use Magento\Eav\Setup\EavSetupFactory;
+use Magento\Eav\Model\Config;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Customer\Model\ResourceModel\Attribute as
+    AttributeResource;
+use Wtc\Film\Model\Attribute\Source\Films;
+
+class AddFavouriteFilm implements DataPatchInterface
+{
+    private ModuleDataSetupInterface $moduleDataSetup;
+
+    private EavSetupFactory $eavSetupFactory;
+
+    private Config $eavConfig;
+
+    private AttributeResource $attributeResource;
+
+    public function __construct
+    (
```

Comment:
The opening bracket for parameters should be on the same line as the method name